types/netmap/netmap.go

```diff
@@ -26,6 +27,7 @@ import (
 // alias parts of previous NetworkMap values.
 type NetworkMap struct {
 	SelfNode   tailcfg.NodeView
+	AllCaps    set.Set[tailcfg.NodeCapability] // set version of SelfNode.Capabilities + SelfNode.CapMap
```

Comment:
Document please that these are only from SelfNode

Reply:
isn't that what this line says on the right hand side after the // ?

Reply:
Oh too long didn't scroll right on phone 🤪